src/Features/Core/Portable/Snippets/SnippetProviders/AbstractSnippetProvider.cs

```diff
@@ -140,7 +141,17 @@ public async Task<SnippetChange> GetSnippetAsync(Document document, int position
                 return null;
             }
 
-            var nodeWithTrivia = node.ReplaceTokens(node.DescendantTokens(descendIntoTrivia: true),
+            var allTokens = node.DescendantTokens(descendIntoTrivia: true).ToList();
+            var filteredTokens = new List<SyntaxToken>();
+
+            // Takes out the first and last token since
+            // those do not need elastic trivia added to them.
+            for (var i = 1; i < allTokens.Count - 1; i++)
+            {
+                filteredTokens.Add(allTokens[i]);
+            }
+
+            var nodeWithTrivia = node.ReplaceTokens(filteredTokens,
```

Comment:
```suggestion
            // Takes out the first and last token since those do not need elastic trivia added to them.
            var nodeWithTrivia = node.ReplaceTokens(allTokens.Skip(1).Take(allTokens.Count - 2),
```

Reply:
or, alternatively, do nothing of `oldToken` is the first or last token of the node.